src/runtime/testing/mock-slot-dom-consumer.ts

```diff
@@ -7,12 +7,15 @@
  * subject to an additional IP rights grant found at
  * http://polymer.github.io/PATENTS.txt
  */
-'use strict';
 
 import {assert} from '../../platform/assert-web.js';
 import {SlotDomConsumer} from '../slot-dom-consumer.js';
 
 export class MockSlotDomConsumer extends SlotDomConsumer {
+  _content;
```

Comment:
maybe add a default value to `_content`, so it doesn't need to be initialized in ctor?

Reply:
If i do that then I have to add explicit types for anything that goes in there.  Maybe later when we get rid of `any` usages?

Reply:
sure, let's do the simplest thing for now.